lib/routes/gmu/yjs.ts

```diff
@@ -0,0 +1,209 @@
+        } else if (type === 'xsgz' && subtype === 'xshd') {
+            // 适配学生活动特殊结构
+            a = element.find('a').first();
+            dateText = element.find('span.time').text().trim() || '';
+            itemTitle = a.text().trim();
+            description = element.find('p').text().trim() || '';
+        } else {
+            // 默认结构
+            a = element.find('h2 a').first();
+            dateText = element.find('h2 span.time').text().trim();
+            itemTitle = a.text().trim();
+            description = element.find('p').text().trim();
+        }
+
+        const href = a.attr('href');
+
+        if (!href || !itemTitle) {
+            return null;
+        }
+
+        const pubDate = parseDate(dateText);
+        if (!pubDate) {
+            return null;
+        }
+
+        let fullLink = href;
+        if (href.startsWith('/')) {
+            fullLink = new URL(href, baseUrl).href;
+        } else if (!href.startsWith('http')) {
+            const currentPath = new URL(link).pathname;
+            const basePath = currentPath.slice(0, Math.max(0, currentPath.lastIndexOf('/') + 1));
+            fullLink = new URL(basePath + href, baseUrl).href;
+        }
+
+        try {
+            const contentResponse = await got(fullLink, {
+                https: {
+                    rejectUnauthorized: false,
+                },
+                timeout: 10000,
+                retry: 3,
+            });
+            const content = load(contentResponse.data);
+
+            // 更新选择器以适应研究生院网站的文章页面结构
+            const articleTitle = content('.winstyle196327 .title').text().trim() || itemTitle;
+            const articleInfo = content('.winstyle196327 .info').text().trim();
+            const articleContent = content('.winstyle196327 .content').html() || description;
+
+            const fullDescription = `
+                <h1>${articleTitle}</h1>
+                <div class="article-info">${articleInfo}</div>
+                <div class="article-content">
+                    ${articleContent}
+                </div>
+                <p><a href="${fullLink}" target="_blank" rel="noopener noreferrer">查看原文</a></p>
+            `;
+
+            return {
+                title: itemTitle,
+                link: fullLink,
+                pubDate,
+                description: fullDescription,
+            };
+        } catch {
+            // 如果获取文章内容失败，返回列表页的简要信息
+            return {
+                title: itemTitle,
+                link: fullLink,
+                pubDate,
+                description,
+            };
+        }
+    }
+
+    const items = await Promise.all(list.toArray().map((item) => processYjsItem(item, type, subtype, baseUrl, link)));
+
+    const validItems = items.filter((item): item is { title: string; link: string; pubDate: Date; description: string } => item !== null);
+
+    if (validItems.length === 0) {
+        throw new Error('No valid items found');
+    }
```

Comment:
Unnecessary check since RSSHub will do it as well
https://github.com/DIYgod/RSSHub/blob/59e7ea876df12bbaf8a4f72d5de12f99f8f9cff3/lib/middleware/parameter.ts#L70-L72